src/services/storage/models/offer.model.ts

```diff
@@ -49,7 +48,7 @@ export default class Offer extends Model {
 
   @Column(DataType.VIRTUAL)
   get availableCapacity (): BigNumber {
-    return this.totalCapacity.minus(this.utilizedCapacity)
+    return new BigNumber(this.totalCapacity).minus(this.utilizedCapacity)
```

Comment:
I think this is a bit hacky. Because we are keeping `utilizedCapacity` as BigNumber but it will even smaller than `totalCapacity`. I suggest also making `utilizedCapacity` a number, and then here you would need to do normal subtraction `this.totalCapacity - this.utilizedCapacity`).

Reply:
Done